docs/source/checks/tabular/train_test_validation/plot_train_test_feature_drift.py

```diff
@@ -156,3 +157,27 @@
 #%%
 # As we see, our condition successfully detects and filters the problematic
 # features that contains a drift!
+#
+# Get an aggregated value
+# =======================
+# Using the reduce_output function we can aggregate the drift values per feature and get a collective score
+# that reflects the effect of the drift across all features on the model. In an unlabeled environment this value
```

Comment:
```suggestion
# that reflects the effect of the drift on the model, taking into account all of the features. When we don't have labels, this value
```

Reply:
unlabeled environment sounds as if "we never get labels"
I wanted the wording to be clear that it's an environment in which we "currently don't have labels" (whether they will or will not arrive). Feel free to change it to something that will emphasize that more...